src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -358,7 +358,7 @@ export const PersonTableRow = ({
                           <Typography color="error">{t('feedback.validation.employments_missing_data')}</Typography>
                         )}
 
-                        <Box sx={{ mt: '1rem' }}>
+                        <Box sx={{ mt: '1rem', padding: '0.5rem' }}>
```

Comment:
```suggestion
                        <Box sx={{ mt: '1rem' }}>
```
Er det en grunn til å legge inn padding her?
![bilde](https://user-images.githubusercontent.com/6313468/203251936-378031f0-a044-4adf-ad8e-7a2e50cb0699.png)